CHANGELOG.md

```diff
@@ -51,13 +52,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Models in `ToolkitTask` with native tool calling no longer need to provide their final answer as `Answer:`.
 - `EventListener.event_types` will now listen on child types of any provided type.
 - Only install Tool dependencies if the Tool provides a `requirements.txt` and the dependencies are not already met.
+- Using `EventListener`s as a context manager now sets thread-local `EventListener`s on the Event Bus instead of adding/removing them.
```

Comment:
Maybe say "Context-local" (or something like that) instead? It's better than thread-local cause contextvars work as expected for coroutines  too (which are in the same thread)

Reply:
Updated wording slightly

Reply:
Could update the PR description too for thoroughness